cli/sql.go

```diff
@@ -49,22 +56,52 @@ func runTerm(cmd *cobra.Command, args []string) {
 		_ = liner.Close()
 	}()
 
+	fmt.Fprint(osStdout, infoMessage)
```

Comment:
I left file support as a todo.
However, stdin already works.
eg:
```bash
$ cat /tmp/test.sql 
SELECT * FROM
system.namespace
where id=
1;
SET database=system; show tables;

$ ./cockroach sql --dev < /tmp/te.sql 
# Welcome to the cockroach SQL interface.
# All statements must be terminated by a semicolon.
# To exit: CTRL + D.
+----------+--------+----+
| parentID |  name  | id |
+----------+--------+----+
| 0        | system | 1  |
+----------+--------+----+
+------------+
|   Table    |
+------------+
| descriptor |
| namespace  |
| users      |
+------------+
```
In this case though, it would be nice to repeat the statements on stdout.